packages/editor-ui/src/components/NodeCredentials.vue

```diff
@@ -93,6 +98,13 @@ export default mixins(
 		...mapGetters('credentials', {
 			credentialOptions: 'allCredentialsByType',
 		}),
+		isProxyCredentialType(): boolean {
+			const proxyCredentialNodeTypes = ["n8n-nodes-base.httpRequest"];
```

Comment:
would need node version. is there possible to have more than one proxy type? should this live in node json instead?

Reply:
HTTP Request node is only one that may access credentials not defined on itself. A simple check for node type should be enough for now.